test/create_env_cluster_topics_test.go

```diff
@@ -15,91 +19,250 @@ import (
-					"confluent_cloud_password":   confluentCloudPassword,
+					"confluent_cloud_api_key":        confluentCloudAPIKey,
+					"confluent_cloud_api_secret":     confluentCloudAPISecret,
+					"google_credentials":             GoogleCredentialsJSON,
+					"confluent_cloud_email":          confluentCloudEmail,
+					"confluent_cloud_password":       confluentCloudPassword,
+					"environment_name":               environmentName,
+					"kafka_cluster_name":             kafkaClusterName,
+					"admin_service_account_name":     adminServiceAccountName,
+					"connector_service_account_name": connectorServiceAccountName,
+					"service_account_name":           topicServiceAccountName,
+					"topic_name_1":                   kafkaTopicName1,
+					"topic_name_2":                   kafkaTopicName2,
+					"bigquery_connector_name":        bigQueryConnectorName,
+					"gcs_connector_name":             gcsConnectorName,
 				},
 			})
 		})
 
 		defer terraform.Destroy(t, runOptions)
 		terraform.InitAndApply(t, runOptions)
 
-		var output string
+		a := assert.New(t)
+
+		// Fetch Terraform outputs
+		envID := terraform.Output(t, runOptions, "environment_id")
+		kafkaClusterID := terraform.Output(t, runOptions, "kafka_cluster_id")
+		bigQueryConnectorID := terraform.Output(t, runOptions, "bigquery_connector_id")
+		gcsSinkConnectorID := terraform.Output(t, runOptions, "connector_gcs_sink_connector_id")
+		topicServiceAccountID := terraform.Output(t, runOptions, "topic_service_account_id")
+		connectorServiceAccountID := terraform.Output(t, runOptions, "connector_service_account_id")
+
+		// Confluent Login, environment, and Kafka cluster setup
+		os.Setenv("CONFLUENT_PLATFORM_PASSWORD", confluentCloudPassword)
+		os.Setenv("CONFLUENT_PLATFORM_EMAIL", confluentCloudEmail)
+		log.Printf("Setting Confluent environment: %s and cluster: %s...", envID, kafkaClusterID)
+		runCLICommand(t, "confluent", "env", "use", envID)
+		runCLICommand(t, "confluent", "kafka", "cluster", "use", kafkaClusterID)
+
+		// Validate Environment name
+		log.Println("Validating environment name...")
+		envOutput := runCLICommand(t, "confluent", "env", "describe", envID, "-o", "json")
+		var envDetails map[string]interface{}
+		a.NoError(json.Unmarshal([]byte(envOutput), &envDetails), "Failed to parse environment details")
+		a.Equal(environmentName, envDetails["name"], "Environment name is not correct.")
+		log.Printf("Validated environment name '%s' successfully.", environmentName)
+
+		// Validate Kafka Cluster Name
+		log.Println("Validating kafka cluster name, status and type...")
+		clusterOutput := runCLICommand(t, "confluent", "kafka", "cluster", "describe", kafkaClusterID, "-o", "json")
+		var clusterDetails map[string]interface{}
+		a.NoError(json.Unmarshal([]byte(clusterOutput), &clusterDetails), "Failed to parse Kafka cluster details.")
+		a.Equal("BASIC", clusterDetails["type"], "Kafka cluster type is not correct, it should be basic.")
```

Comment:
```suggestion
		a.Equal("BASIC", clusterDetails["type"], "Kafka cluster type mismatch.")
```

Reply:
We don't need to specify the expectation here since the [assert function](https://github.com/stretchr/testify/blob/v1.10.0/assert/assertions.go#L460) will handle that

```
if !ObjectsAreEqual(expected, actual) {
		diff := diff(expected, actual)
		expected, actual = formatUnequalValues(expected, actual)
		return Fail(t, fmt.Sprintf("Not equal: \n"+
			"expected: %s\n"+
			"actual  : %s%s", expected, actual, diff), msgAndArgs...)
	}
```